docs/guides/product-vocabulary.mdx

```diff
@@ -0,0 +1,532 @@
+| Join Lawn Care Pro Co\. on Jobber by creating your profile | Join Lawn Care Pro Co\.'s account by creating your own Jobber account |
+
+<br />
+
+### administrator / admin
+
+Always refer to this permission set as **administrator** permission for
+consistency with what appears on **Manage Team**.
+
+<br />
+
+### client / customer
+
+We call our Jobber subscribers **customers** and we call the people they serve
+**clients**.
+
+| **✅ Do **                                              | **❌ Don't**                                              |
+| ------------------------------------------------------- | --------------------------------------------------------- |
+| All selected invoices have been emailed to your clients | All selected invoices have been emailed to your customers |
+
+Sometimes in product, the context makes sense to use **customer** as terms our
+integration partners use or as a blanket term for existing clients and new
+leads.
+
+| **✅ Do **                                           | **❌ Don't**                                           |
+| ---------------------------------------------------- | ------------------------------------------------------ |
+| Sync clients with active QuickBooks Online customers | Sync customers with active QuickBooks Online customers |
+
+<br />
+
+### users / team / team members / field worker
+
+Use **users** when referring to the account's user limit, permissions, or
+individual team management.
+
+| **✅ Do**                                        | **❌ Don't**                                       |
+| ------------------------------------------------ | -------------------------------------------------- |
+| This user will be over your account’s user limit | This person will be over your account's user limit |
+
+Never use **users** to refer to an individual directly.
+
+| **✅ Do**                                                                    | **❌ Don't**                                                                            |
+| ---------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| Feedback from people like you will help us shape the future of this feature. | The feedback we receive from users like you will help shape the future of this feature. |
+
+Use **team member** or team when referring to assignments, time sheets and other
+work specific areas.
+
+| **✅ Do**                                          | **❌ Don't**                                |
+| -------------------------------------------------- | ------------------------------------------- |
+| No team members are currently assigned to this job | No users are currently assigned to this job |
+
+Never use **fieldworker** as this does not cover all company roles
+appropriately.
+
+<br />
+
+### we
```

Comment:
This kind of ties into something lower down about "Contact **our** Success team", is that still OK in this context? We talk about `we` and `you / you're` but `your / our` might be a good pairing to add in here (that was a really confusing sentence to write)

Reply:
I combined `we / our` as it seemed like a natural pairing also but i did add a note to the client hub portion for the pronoun usage (thats the one we seem to go back and forth the most on in Jobber online from my gander)